firmware/Core/Inc/telecommands/telecommand_args_helpers.h

```diff
@@ -6,5 +6,5 @@
 uint8_t TCMD_ascii_to_uint64(const char *str, uint32_t str_len, uint64_t *result);
 uint8_t TCMD_extract_uint64_arg(const char *str, uint32_t str_len, uint8_t arg_index, uint64_t *result);
 uint8_t TCMD_extract_string_arg(const char *str, uint8_t arg_index, char *result, uint16_t result_max_len);
-
+uint8_t TCMD_extract_hex_array_arg(const char *args_str, uint8_t arg_index, uint8_t result_array[], uint16_t result_array_size);
```

Comment:
The more I think about it, I think we should add a `result_length` argument which gets filled to be the number of bytes which get loaded into the `result_array`. You can't currently tell how long the response is, I don't think. Thoughts?